Support for MicoAir405Mini (#26637)
ArduPilot/ardupilot: libraries/AP_HAL_ChibiOS/hwdef/MicoAir405Mini/README.md

# MicoAir405Mini Flight Controller

The MicoAir405Mini is a flight controller designed and produced by [MicoAir Tech.](http://micoair.com/).

## Features

 - STM32F405 microcontroller
 - BMI270 IMU
 - DPS310 barometer
 - AT7456E OSD
 - 9V 2.5A BEC; 5V 2.5A BEC
 - SDCard
 - 6 UARTs
 - 9 PWM outputs

## Physical

![MicoAir F405 Mini V2.1 Front View](MicoAir405Mini_FrontView.jpg)

![MicoAir F405 Mini V2.1 Back View](MicoAir405Mini_BackView.jpg)

## UART Mapping

 - SERIAL0 -> USB
 - SERIAL1 -> UART1 (DMA-enabled)

>  - SERIAL2 -> UART2 (DJI - VTX, TX only is DMA Enabled)

Reviewer: ```suggestion
 - SERIAL2 -> UART2 (DisplayPort, TX only is DMA Enabled)
```

Author: Fixed.

 - SERIAL3 -> UART3 (GPS)

>  - SERIAL4 -> UART4 (TX only is DMA Enabled)

Reviewer: ```suggestion
 - SERIAL4 -> UART4 (MAVLink2,TX only is DMA Enabled)
```

Author: Fixed.

 - SERIAL5 -> UART5 (ESC Telemetry)
 - SERIAL6 -> UART6 (RX6 is inverted from SBUS pin, RX only is DMA Enabled)

## RC Input

> The default RC input is configured on the UART6_RX inverted from the SBUS pin. Other RC  protocols  should be applied at other UART port such as UART1 or UART4, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL6 _Protocol to something other than '23'

Reviewer: ```suggestion
The default RC input is configured as SBUS via the SBUS pin. Other RC  protocols  should be applied at a UART port such as UART1 or UART4, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL6 _Protocol to something other than '23'
```

Author: Fixed.
